src/scss/utils/mixins.scss

```diff
@@ -31,3 +31,121 @@
     text-shadow: none;
   }
 }
+
+/**
+ * This is where all your colours are declared.
+ *
+ * The key should be equivalent to the name of the
+ * class to apply the theme colour, which is in the
+ * format of 'theme-{$element}-{$color}'.
+ *
+ * Classes should be declared on the body so that they
+ * can be accessed and applied to any element.
```

Comment:
> Classes should be declared on the body so that they can be accessed and applied to any element.

Change this to "Classes *are* declared on the body..." since we'll be completing this change.

Reply:
Maybe also mention that these colours should match the $theme_colors configuration applied to SiteConfig.